code/modules/projectiles/guns/energy/special.dm

```diff
@@ -229,26 +225,23 @@ modules/mob/living/carbon/human/life.dm if you die, you will be zoomed out.
 	flags = NODROP
 
 /obj/item/weapon/gun/energy/kinetic_accelerator/Fire()
-	overheat = 1
-	spawn(overheat_time)
-		overheat = 0
-		recent_reload = 0
+	if(!recently_fired)

```

Comment:
For those of you wondering why this is here, simply put, our gun code is shit, so if you can somehow get `Fire` to be called, again, then you would be able to double up on the the amount of reloads you could get (this is what caused the uber fast reload that @TheDZD had to correct for, originally).